src/mixins/styleguide.js

```diff
@@ -1,13 +0,0 @@
-/* eslint-disable */
```

Comment:
@davidyuk why did you remove this file? this breaks the build of the styleguide

Reply:
In 3d4d37fbe80b90f0b8a4e7c0ce91e08e888e6b47 I have also removed this file from `styleguide.config.js`, but @dadaphl reverted it in dd2ab02c4e34048c9a3d977bb1d063262b883916.